bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -417,3 +417,175 @@ def pythonize_rdataframe(klass):
                                       HistoProfileWrapper,
                                       model_class)
         setattr(klass, method_name, getter)
+
+
+from . import pythonization
+from .._numbadeclare import _NumbaDeclareDecorator
```

Comment:
Could you place all (non-repeated) imports at the beginning of the file, together with the ones that are already there?